UmpleToRTCpp/UmpleTLTemplates/Attributes.ump

```diff
@@ -24,6 +24,24 @@ association { 0..1 AbstractGenConstraint constraint -- * GenConstraintCondition
 association { 0..1 AbstractGenConstraint listenerOwner -- * GenConstraintVariable listeners;}
 
 class GenMethod{
+
+    boolean equals(Object object) {
+        if (!(object instanceof GenMethod)) { return false; }
+
+        GenMethod compareTo = (GenMethod)object;
+
+        if (getName() == null && compareTo.getName() != null)
```

Comment:
Can be shortened to a single line:
`return Objects.equals(getName(), compareTo.getName());`

Please test to make sure it works as expected.